school/src/main/java/com/hogwarts/school/controller/FacultyController.java

```diff
@@ -18,16 +19,15 @@ public FacultyController(FacultyService facultyService) {
         this.facultyService = facultyService;
     }
 
-    @PostMapping("/addFaculty")
+    @PostMapping("addFaculty")
     public ResponseEntity<Faculty> addFaculty(@RequestBody Faculty faculty) {
-        Faculty fcl = facultyService.addFaculty(faculty);
-        return ResponseEntity.ok(fcl);
+        return ResponseEntity.ok(facultyService.crateFaculty(faculty));
     }
 
-    @GetMapping("/getFaculty/{facultyId}")
+    @GetMapping("getFaculty/{facultyId}")
```

Comment:
```suggestion
    @GetMapping("/{facultyId}")
```
не нужно ничего писать в пути - это ошибка REST